app/controllers/dashboard/cards_controller.rb

```diff
@@ -1,4 +1,6 @@
 require 'super_memo_service'
+require 'nokogiri'
+require 'open-uri'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.